input/example_metadata.json

```
@@ -1,81 +1,280 @@
+      "path": "crop/example_rotation.json",
+      "type": "json",
+      "properties": "b"
+    },
+    "economy": {
+      "title": "Economy data",
+      "description": "Time is money!",
+      "path": "EEE/example_costs.csv",
+      "type": "csv",
+      "properties": "c"
+    },
+    "emission": {
+      "title": "Emissions data",
+      "description": "Let's think about global warming",
+      "path": "EEE/example_emissions.json",
+      "type": "json",
+      "properties": "d"
+    },
+    "feed": {
+      "title": "Feed data",
+      "description": "Those cows need to eat!",
+      "path": "feed/example_user_feeds.csv",
+      "type": "csv",
+      "properties": "e"
+    },
+    "manure": {
+      "title": "Manure data",
+      "description": "You thought it's a waste, but those crops need to eat as well",
+      "path": "manure/example_manure_management.json",
+      "type": "json",
+      "properties": "f"
+    },
+    "soil": {
+      "title": "Soil data",
+      "description": "Stop treating soil like dirt!",
+      "path": "soil/example_soil.json",
+      "type": "json",
+      "properties": "g"
+    },
+    "weather": {
+      "title": "Weather data",
+      "description": "Let's turn on the AC",
+      "path": "weather/example_weather.csv",
+      "type": "csv",
+      "properties": "h"
+    }
+  },
+  "properties": {
+    "a": {
+      "calf_num": {
+        "type": "number",
+        "description": "The average number of pre-weaned calves (head)",
+        "minimum": 0,
+        "default": 8
+      },
+      "heiferI_num": {
+        "type": "number",
+        "description": "HeiferI is the number of youngstock post-weaning and before breeding (head)",
+        "minimum": 0,
+        "default": 44
+      },
+      "heiferII_num": {
+        "type": "number",
+        "description": "HeiferII is the number of heifers eligible for breeding and in early pregnancy (head)",
+        "minimum": 0,
+        "default": 38
+      },
+      "heiferIII_num_springers": {
+        "type": "number",
+        "description": "HeiferIII is the number of close-up heifers (head)",
+        "minimum": 0,
+        "default": 5
+      },
+      "cow_num": {
+        "type": "number",
+        "description": "The number of dry and lactating cows (head)",
+        "minimum": 0,
+        "default": 100
+      },
+      "herd_num": {
+        "type": "number",
+        "description": "The number of lactating and milking cows on farm (head)",
+        "minimum": 0,
+        "default": 187
+      },
+      "herd_init": {
+        "type": "boolean",
+        "description": "Whether or not to initialize a herd if the milk prod and body weight numbers are exceedingly abnormal",
+        "default": false
+      },
+      "breed": {
+        "type": "string",
+        "description": "The predominant breed of the herd (Holstein - HO or Jersey - JE)",
+        "default": "HO",
+        "minimum_length": 2,
+        "maximum_length": 2
```

```suggestion
        "pattern": "[a-zA-Z]{2}"
```
a more relaxed version would be:
```suggestion
        "pattern": ".{2}"
```